README.md

```diff
@@ -4,8 +4,8 @@
 
```

Comment:
While you're at it, you should change the header of the readme as well to match the repo